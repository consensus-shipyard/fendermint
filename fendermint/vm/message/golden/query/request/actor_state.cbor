a16a4163746f7253746174654a00e2ccdbfcecbdbfbc59