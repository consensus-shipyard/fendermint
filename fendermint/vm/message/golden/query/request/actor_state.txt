ActorState(Address { payload: ID(6447182068707681890) })